copylist.py

```diff
@@ -1,5 +1,22 @@
 #! /usr/bin/env python3
-"""a fast way to copy a list"""
+"""a fast way to make a shallow copy of a list"""
 
 a = [1, 2, 3, 4, 5]
 print(a[:])
+
+"""using the list.copy() method (python3 only)"""
+
+a = [1, 2, 3, 4, 5]
+
+print(a.copy())
```

Comment:
Explicit is better than implicit, the copy method was added in python 3.

On Sun, Mar 22, 2015 at 2:01 PM, Max Brenner <notifications@github.com>
wrote:

> In copylist.py
> <https://github.com/brennerm/PyTricks/pull/25#discussion_r26903322>:
>
> >
> >  a = [1, 2, 3, 4, 5]
> >  print(a[:])
> > +
> > +"""using the list.copy() method (python3 only)"""
> > +
> > +a = [1, 2, 3, 4, 5]
> > +
> > +print(a.copy())
>
> is it really a trick using a copy function?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/brennerm/PyTricks/pull/25/files#r26903322>.
>